### PR TITLE
Add detailed info when throw RejectedRecordsException.

### PR DESCRIPTION
**Pull request by NeoZHL**
*Issue #, if available:*

*Description of changes:*
Add detailed info when throw RejectedRecordsException.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: sample_apps/dotnet/CrudAndSimpleIngestionExample.cs [hunk 1]
```diff
@@ -335,6 +335,11 @@ public async Task WriteRecords()
                 };
                 WriteRecordsResponse response = await writeClient.WriteRecordsAsync(writeRecordsRequest);
                 Console.WriteLine($"Write records status code: {response.HttpStatusCode.ToString()}");
+            } 
+            catch (RejectedRecordsException e) {
+                foreach (RejectedRecord rr in e.RejectedRecords) {
```
Comment (ozanozen) [inline sample_apps/dotnet/CrudAndSimpleIngestionExample.cs:340]: there is no equivalent of System.out.println("RejectedRecords: " + e); here.

File: sample_apps/javaV2/src/main/java/com/amazonaws/services/timestream/CrudAndSimpleIngestionExample.java [hunk 2]
```diff
@@ -172,6 +172,13 @@ public void writeRecords() {
         try {
             WriteRecordsResponse writeRecordsResponse = timestreamWriteClient.writeRecords(writeRecordsRequest);
             System.out.println("WriteRecords Status: " + writeRecordsResponse.sdkHttpResponse().statusCode());
+        } catch (RejectedRecordsException e) {
+            System.out.println("RejectedRecords: ");
```
Comment (ozanozen) [inline sample_apps/javaV2/src/main/java/com/amazonaws/services/timestream/CrudAndSimpleIngestionExample.java:176]: print the exception as well?

File: sample_apps/python/CrudAndSimpleIngestionExample.py [hunk 3]
```diff
@@ -135,6 +135,11 @@ def write_records(self):
             result = self.client.write_records(DatabaseName=Constant.DATABASE_NAME, TableName=Constant.TABLE_NAME,
                                                Records=records, CommonAttributes={})
             print("WriteRecords Status: [%s]" % result['ResponseMetadata']['HTTPStatusCode'])
+        except self.client.exceptions.RejectedRecordsException as err:
+            print("RejectedRecords: ")
```
Comment (ozanozen) [inline sample_apps/python/CrudAndSimpleIngestionExample.py:139]: print the exception as well?

**Review by ozanozen — CHANGES_REQUESTED** [review]
go sample update still seems to be missing.

File: sample_apps/dotnet/CrudAndSimpleIngestionExample.cs (hunk 1 above)
Comment (NeoZHL) [reply sample_apps/dotnet/CrudAndSimpleIngestionExample.cs:341]: > there is no equivalent of System.out.println("RejectedRecords: " + e); here.

Added

**Review by ozanozen — APPROVED** [review]

File: sample_apps/dotnet/CrudAndSimpleIngestionExample.cs [hunk 4]
```diff
@@ -391,6 +397,12 @@ public async Task WriteRecordsWithCommonAttributes()
                 WriteRecordsResponse response = await writeClient.WriteRecordsAsync(writeRecordsRequest);
                 Console.WriteLine($"Write records status code: {response.HttpStatusCode.ToString()}");
             }
+            catch (RejectedRecordsException e) {
+                Console.WriteLine("RejectedRecordsException:" + e.ToString());
+                foreach (RejectedRecord rr in e.RejectedRecords) {
+                    Console.WriteLine("RecordIndex " + rr.RecordIndex + " : " + rr.Reason);
+                }
+            }
```
Comment (zhihaoli) [inline sample_apps/dotnet/CrudAndSimpleIngestionExample.cs:405]: Should we have the equivalent of System.out.println("Other records were written successfully. "); here in dotnet and below in the js app?

File: sample_apps/python/CrudAndSimpleIngestionExample.py [hunk 5]
```diff
@@ -135,6 +135,11 @@ def write_records(self):
             result = self.client.write_records(DatabaseName=Constant.DATABASE_NAME, TableName=Constant.TABLE_NAME,
                                                Records=records, CommonAttributes={})
             print("WriteRecords Status: [%s]" % result['ResponseMetadata']['HTTPStatusCode'])
+        except self.client.exceptions.RejectedRecordsException as err:
+            print("RejectedRecords: ", err)
+            for rr in err.response["RejectedRecords"]:
+                print("Rejected Index " + str(rr["RecordIndex"]) + ": " + rr["Reason"])
+            print("Other records have already been wrote successfully. ")
```
Comment (zhihaoli) [inline sample_apps/python/CrudAndSimpleIngestionExample.py:142]: can you update this to "Other records were written successfully. "

File: sample_apps/python/CrudAndSimpleIngestionExample.py [hunk 6]
```diff
@@ -170,6 +175,11 @@ def write_records_with_common_attributes(self):
             result = self.client.write_records(DatabaseName=Constant.DATABASE_NAME, TableName=Constant.TABLE_NAME,
                                                Records=records, CommonAttributes=common_attributes)
             print("WriteRecords Status: [%s]" % result['ResponseMetadata']['HTTPStatusCode'])
+        except self.client.exceptions.RejectedRecordsException as err:
+            print("RejectedRecords: ", err)
+            for rr in err.response["RejectedRecords"]:
+                print("Rejected Index " + str(rr["RecordIndex"]) + ": " + rr["Reason"])
+            print("Other records have already been wrote successfully. ")
```
Comment (zhihaoli) [inline sample_apps/python/CrudAndSimpleIngestionExample.py:182]: can you update this to "Other records were written successfully. "

**Review by zhihaoli — COMMENTED** [review]
Do we need to update the go app as well?

**Review by zhihaoli — APPROVED** [review]

**Pull request merged**